Using websocket with Node-RED (#2672)
FlowFuse/website: src/node-red/protocol/websocket.md

---
eleventyNavigation:
  key: Websocket
  parent: "Communication Protocols"
meta:
   title: Using Websocket with Node-RED
   description: 
   keywords: node-red websocket, websocket in node-red, websocket node-red
image: 
---

# {{meta.title}}

> In this documentation, you will learn how to use WebSocket in Node-RED for communication. We will explain the concepts through practical examples, covering how to connect and communicate as a WebSocket client and how to build a WebSocket server using Node-RED to communicate with clients.

Reviewer: ```suggestion
Let's look at how to use WebSocket in Node-RED for communication. We will explain the concepts through practical examples, covering how to connect and communicate as a WebSocket client and how to build a WebSocket server using Node-RED to communicate with clients.
```
